Dockerfile

```diff
@@ -0,0 +1,18 @@
+# syntax=docker/dockerfile:1
+
+FROM golang:1.21 as build
+
+COPY . /root
+WORKDIR /root
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux go build -o /durabletask-go
+
+FROM gcr.io/distroless/static-debian11
```

Comment:
```suggestion
FROM gcr.io/distroless/static-debian12:nonroot
```

Reply:
There's a newer version and also we should use the "nonroot" variant if possible

Reply:
updated!